packages/fork-choice/src/protoArray/protoArray.ts

```diff
@@ -299,12 +299,14 @@ export class ProtoArray {
          *     ii) lazy: that invalidation was result of simple check and the EL just
          *         responded with a bogus LVH
          *
-         *   So we will just invalidate the current payload and let future responses take care
-         *   to be as robust as possible.
+         *   In such case for robustness, lets not process this invalidation into forkchoice
+         *   as it might poision it since the invalidations can't be processed unless latestValidHashIndex
+         *   is known as invalidateFromParentIndex is the parent of the payload being verified which has not
+         *   been imported yet into forkchoice.
```

Comment:
Can you replace this comment block?
```
         * The LVH (latest valid hash) is null or not found.
         *
         * The spec gives an allowance for the EL being able to return a nullish LVH if it could not "determine" one.
         * There are two interpretations:
         * - "the LVH is unknown" - simply throw and move on. We can't determine which chain to invalidate since we don't know which ancestor is valid.
         * - "the LVH doesn't exist" - this means that the entire ancestor chain is invalid, and should be marked as such.
         * 
         * The more robust approach is to treat nullish LVH as "the LVH is unknown" rather than "the LVH doesn't exist".
         * The alternative means that we will poison a valid chain when the EL is lazy (or buggy) with its LVH response.
```